jetty-server/src/main/config/modules/secure-redirect.mod

```diff
@@ -0,0 +1,17 @@
+DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Enable SecuredRedirectHandler to redirect all http requests to https.
```

Comment:
Maybe add some note here that it redirects to the secure port set in the ???? module.